lib/transaction/operation_congress_voting_result.go

```diff
@@ -0,0 +1,70 @@
+package transaction
+
+import (
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/error"
+	"encoding/json"
+	"net/url"
+)
+
+type OperationBodyCongressVotingResult struct {
+	BallotStamps struct {
+		Hash string
+		Urls []string
+	}
+	Voters struct {
+		Hash string
+		Urls []string
+	}
+	Result struct {
+		Count uint64
+		Yes   uint64
+		No    uint64
+		ABS   uint64
+	}
+}
+
+func (o OperationBodyCongressVotingResult) Serialize() (encoded []byte, err error) {
+	return json.Marshal(o)
+}
+func (o OperationBodyCongressVotingResult) IsWellFormed([]byte) (err error) {
+	if len(o.BallotStamps.Hash) == 0 {
+		return errors.ErrorOperationBodyInsufficient
+	}
+
+	var tobeHash = ""
+	for _, u := range o.BallotStamps.Urls {
+		if _, err := url.Parse(u); err != nil {
+			return errors.ErrorInvalidOperation
+		}
+		tobeHash += u
+	}
+
+	hash := common.MakeHash([]byte(tobeHash))
```

Comment:
@kfangw When I introduced the first thought of voting result things, the `Hash` is the file checksum of voting result, the file is the downloaded from `Urls`.

Reply:
Then, the `Hash` is not verifiable, right?
@spikeekips

Reply:
@kfangw Yeah, I thought so. I can be verifiable by manual.